src/test/java/com/jalasoft/sdfc/steps/CommonSteps.java

```diff
@@ -87,4 +87,16 @@ public void iValidateTheOpportunityIsNotInTheMenu(String name, String menuName)
         navBar.clickOnTabNameArrow(Item.valueOfItem(menuName));
         assertFalse(navBarMenu.isMenuItemVisible(name));
     }
+
+    @Then("I validate the create form an error message says {string}")
```

Comment:
I would change this to `I verify the following error message in the form: {string}`

Reply:
updated